packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -628,12 +628,12 @@ const ReleaseDetailsBody = () => {
                           entry.contentType.mainFieldValue || entry.id
                         }`}</Typography>
                       </Td>
-                      <Td>
+                      <Td width={'15%'}>
```

Comment:
These can all be strings directly:

```suggestion
                      <Td width='15%'>
```

Reply:
Shouldn't each Td have a percentage and shouldn't it all add up to 100%?

Reply:
I'm also wondering if we could use [flex-grow](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow) instead of percentages, have you considered or could you see if it could work for us here?

Reply:
Flex grow doesn t work because Tr are not Flex, so I need also to rework and restyle the Tr if we want to use it